ci-jobs/generate-docs.yml

```diff
@@ -1,40 +0,0 @@
-# parameters: # No params yet, but may add later
```

Comment:
this one is only used in 1.x branch by https://dev.azure.com/AppiumCI/Appium%20CI/_build?definitionId=38&_a=summary , so we can remove this safely from current master